external-import/lia-file-feed/docker-compose.yml

```
@@ -1,7 +1,7 @@
 version: '3'
 services:
-  liafilefeed:
-    image: opencti/liafilefeed:6.4.1
+  lia-file-feed:
+    image: opencti/lia-file-feed:6.4.1
```

Add the word "connector" to the image
```suggestion
    image: opencti/connector-lia-file-feed:6.4.1
```

---

Hi @LIA-Intel, please be aware that Docker image's name and version have been updated 🙏